### ISSUE TITLE
make error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#1]
```
What steps will reproduce the problem?
1. hg clone https://pino-twitter.googlecode.com/hg/ pino-twitter
2. ./waf configure
3. ./waf make

What do you see instead?

default/src/main_window.c: In function ‘_lambda20_’:
default/src/main_window.c:748: error: ‘_data1_’ undeclared (first use in
this function)
default/src/main_window.c:748: error: (Each undeclared identifier is
reported only once
default/src/main_window.c:748: error: for each function it appears in.)
default/src/main_window.c:754: warning: passing argument 2 of
‘gtk_about_dialog_set_authors’ from incompatible pointer type
/usr/include/gtk-2.0/gtk/gtkaboutdialog.h:103: note: expected ‘const gchar
**’ but argument is of type ‘char **’
default/src/main_window.c: In function ‘main_window_link_clicking’:
default/src/main_window.c:1120: warning: format not a string literal and no
format arguments
Waf: Leaving directory `/home/marius/Desktop/pino-twitter/_build_'
Build failed
 -> task failed (err #1): 
    {task: cc main_window.c -> main_window_1.o}

What version of the product are you using? On what operating system?

Mercurial version on Ubuntu 9.10

Please provide any additional information below.

I have all the required dependencies .
```

Original issue reported on code.google.com by `marius.c...@gmail.com` on 30 Nov 2009 at 9:22

**Comment by GoogleCodeExporter** [comment]
```
What version of vala do you have installed? Your Ubuntu has last updates? I 
can't 
reproduce this bug.
```

Original comment by `tro...@gmail.com` on 30 Nov 2009 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
I have valac 0.7.7-3-ppa6

My system is always up to date :)
```

Original comment by `marius.c...@gmail.com` on 30 Nov 2009 at 11:16

**Comment by GoogleCodeExporter** [comment]
```
Try to downgrade your vala to distributive version (0.7.6). This is the first 
thing 
that comes to mind.
```

Original comment by `tro...@gmail.com` on 30 Nov 2009 at 11:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 30 Nov 2009 at 3:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Got it to build removing this line 755 
default/src/main_window.c
 _data2_->_data1_ = block1_data_ref (_data1_);

```

Original comment by `lxga...@gmail.com` on 13 Dec 2009 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
I have the same error with pino 0.1.0b4 on Arch Linux.

vala 0.7.8
gcc 4.4.2

```

Original comment by `pizzap...@gmail.com` on 19 Dec 2009 at 12:23

**Comment by GoogleCodeExporter** [comment]
```
I think that's a Vala issue, however, here is a patch for pino 0.1.0rc1 that
circumvents the issue.
```

Original comment by `pizzap...@gmail.com` on 21 Dec 2009 at 1:57

Attachments:
- [pino_aboutdialog.patch](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-1/comment-8/pino_aboutdialog.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot.

Closed by 2b642ab481 revision.
```

Original comment by `tro...@gmail.com` on 21 Dec 2009 at 2:47
- Changed state: **Fixed**
